frontend/apps/crates/entry/user/src/register/pages/step_2/dom.rs

```diff
@@ -38,9 +40,9 @@ impl Step2Page {
                         Some(STR_LANGUAGE_LABEL),
                         Some(STR_LANGUAGE_PLACEHOLDER),
                         None,
-                        STR_LANGUAGE_OPTIONS.to_vec(),
+                        languages::EMAIL_LANGUAGES.iter().map(|l| Language::from(l.clone())).collect(),
```

Comment:
Convert into the wrapper for `utils::languages::Language`

Reply:
Converting from one type of `Language` to another feels kind of weird but I don't see a way around it, so I guess it's fine

Reply:
I'm not super happy about using it either. I tried going with having the lists convert to a hashmap and SimpleSelect using Entry but it made the SimpleSelect quite a bit more complex. We could look into refactoring a bit at a later stage to improve it.